blob/.npmignore

```diff
@@ -1,5 +1,5 @@
-!dist/lib/**/*.js
-!es6dist/lib/**/*.js
+!dist/**/*.js
+!dist-esm/lib/**/*.js
```

Comment:
There are other `es6dist` in this npmignore file

Reply:
Fixed.